examples/private_connection/README.md

```diff
@@ -0,0 +1,61 @@
+# Private Connectivity Example
+
+This example shows how to configure a forseti to use only private connection between server, client and GCP services (e.g. Storage or Cloud SQL).
```

Comment:
This example shows how to configure a Forseti instance to use only private connections between its components (Server VM, Client VM and CloudSQL instance) and limit public egress traffic to only [Google private VIP](https://cloud.google.com/vpc/docs/configure-private-google-access#private-domains) post installation. Note that egress traffic to the public internet is still needed during Forseti installation phase in order to install necessary deb package dependencies.

Reply:
done